packages/@o3r/analytics/package.json

```diff
@@ -96,6 +97,8 @@
     "jest-preset-angular": "~13.1.1",
     "nx": "~16.6.0",
     "rxjs": "^7.8.1",
+    "semver": "^7.5.2",
+    "ts-jest": "~29.1.1",
```

Comment:
should not be needed after rebase

Reply:
it is still used, should I switch from `ts-jest`  preset to `jest-preset-angular` ?

Reply:
the only dependency on Angular is for the schematics for ng-add so I'll keep `ts-jest`